test/verbs.js

```diff
@@ -47,44 +64,84 @@ describe('http verbs', function () {
       .set('Content-Type', 'application/x-www-form-urlencoded')
       .send('mypost=hello')
       .end(function (err, res) {
-        assert.equal(JSON.stringify(res.body.form), '{"mypost":"hello"}');
-        done(err);
+        if (err) { return done(err); }
+        assert.equal(res.text, 'mypost=hello');
+        done();
```

Comment:
I also like this less. Let's see if we can make that look more like the original text